### PR TITLE
Switch the solution's minimum version to VS2015

### LINKED ISSUE #615: Switch the solution's minimum version to VS2015
From https://github.com/FakeItEasy/FakeItEasy/issues/537#issuecomment-188925534

Would allow us to use fancy C#6 features. Also supports other issues, like #489.

### PR DESCRIPTION
**Pull request by thomaslevesque**
Fixes #615

### CODE REVIEW

File: FakeItEasy.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 14
```
Comment (blairconrad) [inline FakeItEasy.sln]: Is it called "14" or should we have VS2015 here? Or did the tools choose this name?

File: rakefile.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 require 'albacore'
 
-msbuild_command = [
-  "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe",
-  "C:/Program Files (x86)/MSBuild/12.0/Bin/MSBuild.exe"
-].find(lambda{ raise "MSBuild not found"}) { |msbuild| File.file?(msbuild) }
+msbuild_command = "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe"
+if !File.file?(msbuild_command)
+  raise "MSBuild not found"
+end
```
Comment (blairconrad) [inline rakefile.rb]: And I was so proud of having learned the fancy 'find in array or blow up' construction. Ah well.

**Comment by blairconrad** [comment]
It looks very good to me, except for my minor question about "VS 14/VS 2015". Assuming we like "14", I don't see a reason not to merge before 2.0. Does anyone else?

File: FakeItEasy.sln (hunk 1 above)
Comment (thomaslevesque) [inline FakeItEasy.sln]: Yes, that's weird. But I didn't choose to put "14" there, Visual Studio did ;)

File: rakefile.rb (hunk 2 above)
Comment (adamralph) [inline rakefile.rb]: We'll probably want it back once VS 15 goes RTM :wink:

**Comment by adamralph** [comment]
"Visual Studio 14" is indeed chosen by VS when saving the solution file.

**Comment by thomaslevesque** [comment]
Thanks for the merge @adamralph

**Pull request merged**